### DISCUSSION TITLE
New Trading

### DISCUSSION THREAD

**Discussion by Dionis404** [sunflower-land/sunflower-land#3434] [Ideas]
@Rofrtd I hear you're working on improving trading, so here's my idea.  

Remove the "ball" in the goblin village. Prohibit trading at Plaza and Beach. Create a trading quarter with new NPCs.

### Bazaar
A bumpkin stands in the trading quarter with a displayed lot and waits for it to be bought from him.
The commission is the same as now: 1 BB per transaction. But there are limitations, no more than N transactions per day/week (this parameter may be tied to the bumpkin's level).

### Shop (Main type)
The player builds a shop on their farm, and to make it function, they need to pay a trading license (BB, SFL, or possibly resources).
The number of lots in the shop is limited and depends on its level; the player will need to spend resources to upgrade their shop.

**How to sell?**

The player enters the shop and selects the resource, quantity, and price per unit. They also choose how this lot will be sold, either individually or in batches.

What's the advantage of selling resources individually? At the start of the game, obtaining most resources costs more than buying them on the market. However, buying 1 gold or several irons is extremely difficult in the game itself. Yes, there is an opportunity to buy them at www.sequence.market, but in this case, whales will simply continue to increase their resource reserves without removing them from the game.

**How to buy?**
The player comes to the trading quarter, approaches an NPC, and sees a list of all resources. They choose the one they need and sort it by the parameters they want. After finding a profitable offer, they press the key to go to the shop and end up at the seller's farm. They buy the goods and can return to their farm or back to the trading quarter.

**The ability to place a request for sale.**
A player with a shop can place a lot to buy the resource they need.
This scheme allows new players to sell resources without having a shop and without standing on the plaza for hours hoping someone will buy something.

Improving trading should be aimed at making "whales" more willing to sell their resources, and making it easier for new players to buy those resources.

**Comment by brian-man** [comment]
By default I'm against any proposal that is "take something away for my new shiny thing that nobody has designed, built, tested, debugged, or used".

Your new feature should stand on its own without requiring a forced monopoly.

**Comment by rasheed19871** [comment]
What I want to see is simple and unique I think there should be a modern live board (meaning no need to click on it just stay visually open and update in real time) with the ability to filter will be Amazing
